src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { getUserById } from './utils/getUserById';
+
+export const App: React.FC = () => {
+  const preparedTodos: Todo[] = todosFromServer.map(todo => {
+    return {
+      ...todo,
+      user: getUserById(todo.userId) || null,
+    };
+  });
+
+  const [todo, setTodo] = useState<Todo[]>(preparedTodos);
```

Comment:
[CODE STYLE] - The variable name 'todo' is a bit misleading here as it represents an array of todos, not a single todo. Consider renaming it to 'todos' to reflect that it's a collection.